hdfs-executor/src/main/java/org/apache/mesos/hdfs/executor/AbstractNodeExecutor.java

```diff
@@ -198,6 +199,9 @@ protected void startProcess(ExecutorDriver driver, Task task) {
     if (task.getProcess() == null) {
       try {
         ProcessBuilder processBuilder = new ProcessBuilder("sh", "-c", task.getCmd());
```

Comment:
I started there... and that didn't work.  I'll take another look since I don't remember the details of the moment.